README.md

```diff
@@ -18,14 +18,22 @@ juju deploy identity-platform-login-ui-operator
 To set the Ory Kratos endpoint to the UI server, do:
 
 ```console
-juju config identity-platform-login-ui-operator kratos_url={kratos_url:port}
+juju relate identity-platform-login-ui-operator kratos
 ```
 
 To set the Ory Hydra endpoint to the UI server, do:
 
 ```console
-juju config identity-platform-login-ui-operator hydra_url={hydra_url:port}
+juju relate identity-platform-login-ui-operator hydra
```

Comment:
I think you'll need to specify relation names when multiple integrations are in place for the 2 charms.
We could also simplify the `Interacting with Kratos and Hydra` part by splitting it into 2 parts - for Hydra and Kratos. For example:
### Hydra
This charm offers integration with `hydra-operator`. To integrate them, run:
```bash
juju integrate hydra:hydra-endpoint-info identity-platform-login-ui-operator:hydra-endpoint-info
juju integrate hydra:ui-endpoint-info identity-platform-login-ui-operator:ui-endpoint-info
```
### Kratos
This charm offers integration with `kratos-operator`. To integrate them, run:
```bash
juju integrate kratos:kratos-endpoint-info identity-platform-login-ui-operator:kratos-endpoint-info
juju integrate kratos:ui-endpoint-info identity-platform-login-ui-operator:ui-endpoint-info
```
What do you think?